sdk_contrib/async_processor/src/test/java/io/opentelemetry/sdk/contrib/trace/export/DisruptorAsyncSpanProcessorTest.java

```diff
@@ -268,4 +272,26 @@ public void multipleForceFlush() {
     assertThat(incrementSpanProcessor.getCounterOnExportedForceFlushSpans()).isEqualTo(tenK);
     assertThat(incrementSpanProcessor.getCounterOnShutdown()).isEqualTo(1);
   }
+
+  abstract static class ConfigBuilderTest extends ConfigBuilder<ConfigBuilderTest> {
+    public static NamingConvention getNaming() {
+      return NamingConvention.DOT;
+    }
+  }
+
+  @Test
+  public void configTest() {
```

Comment:
We need a test like this for the zipkin config, as well.

Reply:
Oh thank you! I totally miss it! :facepalm: